### PR TITLE
Set hostname when dhcp provides it

### PR DESCRIPTION
**Pull request by grimpy**
Signed-off-by: Jo De Boeck <deboeck.jo@gmail.com>

### CODE REVIEW

File: config/udhcp/simple.script [hunk 1]
```diff
@@ -33,6 +33,9 @@ case "$1" in
 				: $(( metric += 1 ))
 			done
 		fi
+        if [ -n "$hostname" ] ; then
```
Comment (maxux) [inline config/udhcp/simple.script]: Wrong indentation

Comment (grimpy) [inline config/udhcp/simple.script]: oops must be a tabs vs spaces thing let me check

File: config/udhcp/simple.script [hunk 2]
```diff
@@ -33,6 +33,9 @@ case "$1" in
...
+            hostname "$hostname"
+        fi
```
Comment (maxux) [inline config/udhcp/simple.script]: No need to adapt /etc/hosts too ?

Comment (grimpy) [inline config/udhcp/simple.script]: ideally yes, but how would we approach this constantly overwrite it? Grep if hostname exists in it and add if not?

Comment (maxux) [inline config/udhcp/simple.script]: Maybe for a first version, just sed /d the 127.0.0.1 and ::1 lines and rewrite them

**Pull request merged**